modepy/shapes.py

```diff
@@ -0,0 +1,547 @@
+# {{{ docstring
+
+r"""
+:mod:`modepy.shapes` provides a generic description of the supported shapes
+(i.e. reference elements).
+
+.. currentmodule:: modepy
+
+.. autoclass:: Shape
+.. autoclass:: Face
+.. autofunction:: biunit_vertices_for_shape
```

Comment:
I'm tempted to push for `biunit_vertices_for_shape`. Everywhere else calls "bi-unit" just unit. No really good reason to start being inconsistent with that now, even if it's technically more precise. What do you think?

Reply:
Do you mean `unit_vertices_for_shape`?

From a quick grep, only `unit_to_barycentric` seems to still be using that + some documentation that can be updated, so it doesn't seem like too big of a hassle to change everything to `biunit`.

There's a lot of `unit_nodes` in `meshmode` though, so I'm leaning more towards just using `unit_vertices_for_shape`, since the coordinates are documented.

Reply:
Agreed. My thought to go back to `unit` instead of `biunit` came after some meshmode hacking. I don't/no longer think it's worth it to start being inconsistent with all that code.

Reply:
Renamed in 338e1aa.